packages/matchbox/src/components/Grid/styles.js

```diff
@@ -0,0 +1,25 @@
+import { tokens } from '@sparkpost/design-tokens';
+
+const gutterWidth = tokens.spacing_400;
```

Comment:
This was the closest token to the original gutter width, `rem(18)`.

<img width="1388" alt="Screen Shot 2020-04-02 at 2 50 35 PM" src="https://user-images.githubusercontent.com/1335605/78290555-dfdd8b00-74f1-11ea-9697-589fdc8da2bf.png">